plugin/fugitive_pagure.py

```diff
@@ -23,9 +23,15 @@ def is_pagure(remote):
     domains = ["pagure.io", "src.fedoraproject.org"]
     return any(d in remote for d in domains)
 
+def is_fork(remote):
+    token = remote.rsplit("/")[3]
+    return (token == 'forks')
 
 def remote2http(remote):
-    url = remote
```

Comment:
If you keep this line, then you can drop the `else` branch.

Reply:
Alrighty :+1: